extra/twig-extra-bundle/Resources/config/markdown_league.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use League\CommonMark\CommonMarkConverter;
+use Twig\Extra\Markdown\LeagueMarkdown;
+use Twig\Extra\TwigExtraBundle\LeagueCommonMarkConverterFactory;
+
+return static function (ContainerConfigurator $container) {
+    $service = \function_exists('Symfony\Component\DependencyInjection\Loader\Configurator\service') ? 'Symfony\Component\DependencyInjection\Loader\Configurator\service' : 'Symfony\Component\DependencyInjection\Loader\Configurator\ref';
+    $container->services()
+        ->set('twig.markdown.league_common_mark_converter_factory', LeagueCommonMarkConverterFactory::class)
+        ->args([tagged_iterator('twig.markdown.league_extension')])
```

Comment:
it looks like `tagged_iterator` did not exist in SF 4.3

Reply:
The dev deps don't allow installing 4.3 but I switched to `TaggedIteratorArgument` to be safe.